src/kernel/hartlock.c

```diff
@@ -0,0 +1,30 @@
+#include "hart_locals.h"
```

Comment:
use `#include <>`, not `#include ""` (and same for all files below)

Reply:
(also, plz clang-format everything... we should probably have a makefile rule that just does that, feel free to make me add that if we're working on this tmrw)